code/components/citizen-server-impl/include/state/SyncTrees_RDR3.h

```diff
@@ -367,7 +403,20 @@ struct CDoorScriptGameStateDataNode { bool Parse(SyncParseState& state) { return
 struct CHeliHealthDataNode { bool Parse(SyncParseState& state) { return true; } };
 struct CHeliControlDataNode { bool Parse(SyncParseState& state) { return true; } };
 
-struct CObjectCreationDataNode { bool Parse(SyncParseState& state) { return true; } };
+struct CObjectCreationDataNode  : GenericSerializeDataNode<CObjectCreationDataNode>
+{
+	uint32_t m_model;
+	bool m_hasInitPhysics;
+
+	template<typename Serializer>
+	bool Serialize(Serializer& s)
+	{
+		s.Serialize(32, m_model);
```

Comment:
This part is invalid. First field is a bool. It seems model hash is serialized somewhere later in the data node.